ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/HARouting.java

```diff
@@ -361,24 +385,23 @@ private static void forwardTo(
               shouldCancelRequests
           );
     } catch (Exception e) {
-      // If we threw some explicit exception, then let it bubble up. All of the row handling is
-      // wrapped in a KsqlException, so any intentional exception or bug will be surfaced.
-      final KsqlException ksqlException = causedByKsqlException(e);
-      if (ksqlException != null) {
-        throw ksqlException;
-      }
       // If the exception was caused by closing the connection, we consider this intentional and
       // just return.
       if (shouldCancelRequests.isDone()) {
         LOG.warn("Connection canceled, so returning");
         return;
       }
+
```

Comment:
All exceptions thrown by a host that a query was forwarded to, will result in a `StandbyFallbackException` so that we can keep trying other standbys. Previously, we were throwing a `KsqlException` which was preventing us from keep trying other hosts